templates/deno/app/entry.server.tsx

```diff
@@ -1,16 +1,16 @@
-import React from "react";
-import { renderToString } from "react-dom/server";
-import { RemixServer } from "@remix-run/react";
 import type { EntryContext } from "@remix-run/deno";
+import { RemixServer } from "@remix-run/react";
+import * as React from "react";
+import { renderToString } from "react-dom/server";
 
 export default function handleRequest(
   request: Request,
   responseStatusCode: number,
   responseHeaders: Headers,
-  remixContext: EntryContext,
+  remixContext: EntryContext
 ) {
   const markup = renderToString(
-    <RemixServer context={remixContext} url={request.url} />,
+    <RemixServer context={remixContext} url={request.url} />
```

Comment:
We use `deno lint` and `deno fmt` for Deno code, so we should not remove the trailing `,`

Reply:
Good to know! I agree that it'd be nice to ignore `prettier` here. In any case, these changes shouldn't be included in the PR.

Reply:
Seems like it's not ignored by default because we use the root `.eslintignore` file as ignore file in our `format` script.